redbot/core/errors.py

```diff
@@ -49,3 +49,9 @@ def __str__(self) -> str:
         return _("{user}'s balance cannot rise above {max:,} {currency}.").format(
             user=self.user, max=self.max_balance, currency=self.currency_name
         )
+
+
+class BankPruneError(BankError):
+    """Raised when trying to prune a local bank and no server is specified."""
+
```

Comment:
Does Black yell without a new line here?

Reply:
Yeah it requires 2 blank lines between classes